irictl-machine/cmd/irictl-machine/irictlmachine/get/event/event.go

```diff
@@ -68,11 +73,16 @@ func Run(
 	render renderer.Renderer,
 	opts Options,
 ) error {
-	var filter *iri.EventFilter
+	var filter *iri.EventFilter = &iri.EventFilter{
+		EventsFromTime: time.Now().Add(-1 * DefaultDuration).Unix(),
+		EventsToTime:   time.Now().Unix(),
+	}
 	if opts.Labels != nil {
-		filter = &iri.EventFilter{
-			LabelSelector: opts.Labels,
-		}
+		filter.LabelSelector = opts.Labels
+	}
+
+	if opts.Duration > 0 {
+		filter.EventsFromTime = time.Now().Add(-1 * opts.Duration).Unix()
 	}
```

Comment:
This is not needed if Options.Duration always contains the actual duration

Reply:
removed check